### PR TITLE
Create a JoliNotifier based on JoliNotif

### PR DESCRIPTION
**Pull request by pyrech**
I added the dependency in the composer's suggest instead of require section because JoliNotif is PHP 5.4+

### CODE REVIEW

File: src/Sismo/Notifier/JoliNotifier.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+/*
+ * This file is part of the Sismo utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sismo\Notifier;
```
Comment (lyrixx) [inline src/Sismo/Notifier/JoliNotifier.php]: Can you move it to the Contrib NS ?

Comment (pyrech) [inline src/Sismo/Notifier/JoliNotifier.php]: Done

File: src/Sismo/Contrib/JoliNotifier.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ */
+class JoliNotifier extends Notifier
+{
+    private $format;
+    private $notifier;
+
+    public function __construct($format = "[%STATUS%]\n%message%\n%author%")
+    {
+        $this->format   = $format;
+        $this->notifier = NotifierFactory::create();
```
Comment (lyrixx) [inline src/Sismo/Contrib/JoliNotifier.php]: May be, you should add something like:

```
if (!class_exists('....')) throw new RuntimeException('the package .... is needed...')
```

to help bugging.

Comment (pyrech) [inline src/Sismo/Contrib/JoliNotifier.php]: Yes, you're right

Comment (pyrech) [inline src/Sismo/Contrib/JoliNotifier.php]: Done

**Comment by lyrixx** [comment]
Thanks for your work on this new feature!

**Pull request merged**